src/playground/index.ejs

```diff
@@ -2,6 +2,17 @@
 <html>
   <head>
     <meta charset="UTF-8">
+
+    <meta name="twitter:card" content="summary"/>
+    <meta name="twitter:site" content="Programmieren mit der Maus"/>
+    <meta name="twitter:creator" content="wdrmaus.de"/>
+    <meta property="og:url" content="https://programmieren.wdrmaus.de"/>
+    <meta property="og:title" content="Programmieren mit der Maus"/>
+    <meta property="og:description" content="Hier lernen Kinder Bildergeschichten und Spiele zu programmieren."/>
+    <meta property="og:image" content="<%=require('../../assets/img/head_logo.png')%>"/>
+    <meta name="twitter:site" content="@DieMaus"/>
+    <meta name="twitter:image:src" content="<%=require('../../assets/img/head_logo.png')%>" />
```

Comment:
Seems like the url needs to be absolute, so you should prepend `https://programmieren.wdrmaus.de` to this (and the `og:image` thingy)